Sources/MountebankSwift/Models/Stub/Response/Behavior.Copy.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+extension Behavior {
+    public struct Copy: Equatable, Codable {
+        public struct Using: Equatable, Codable {
+            public enum Method: String, Equatable, Codable {
+                case regex
+                case xpath
+                case jsonpath
+            }
+
+            public struct Options: Equatable, Codable {
```

Comment:
Would it be swifty to make this an OptionSet?

Reply:
Created a OptionSet in 67064d4c3b9d61607d4338a8b2228fa9122d6ff0